index.js

```diff
@@ -0,0 +1,41 @@
+import Books from './modules/booksclass.js';
```

Comment:
- [ ] The source of this file is missing. Please fix this issue before requesting the next review

Reply:
Hello @jasonfelice . A friend checked the code in his computer and says he has the same problem, that the module folder is empty. But in my computer it is fine. I'm not sure how to fix this without creating a new repo. Could you give me more insight into this problem please?